flink-table/flink-sql-client/src/test/resources/sql/set.q

```diff
@@ -167,6 +167,37 @@ execution.shutdown-on-attached-exit=false
 execution.target=remote
 jobmanager.rpc.address=$VAR_JOBMANAGER_RPC_ADDRESS
 pipeline.classpaths=
-pipeline.jars=$VAR_PIPELINE_JARS
+pipeline.jars=
+rest.port=$VAR_REST_PORT
+!ok
+
+# test add jar
+ADD JAR '$VAR_UDF_JAR_PATH';
+[INFO] The specified jar is added into session classloader.
+!info
+
+set;
+execution.attached=true
+execution.savepoint.ignore-unclaimed-state=false
+execution.shutdown-on-attached-exit=false
+execution.target=remote
+jobmanager.rpc.address=localhost
+pipeline.classpaths=
+pipeline.jars=$VAR_PIPELINE_JARS_URL
+rest.port=$VAR_REST_PORT
+!ok
+
+reset;
+[INFO] All session properties have been set to their default values.
+!info
```

Comment:
Would be better to call `create function func1 as 'LowerUDF' LANGUAGE JAVA;` and `SELECT id, func1(str) FROM (VALUES (1, 'Hello World');` to verify the added jar still works.